pkg/server/plugin/nodeattestor/aws/iid.go

```diff
@@ -143,7 +119,12 @@ func (p *IIDAttestorPlugin) Attest(stream nodeattestorv1.NodeAttestor_AttestServ
 		return status.Error(codes.InvalidArgument, "missing attestation payload")
 	}
 
-	attestationData, err := unmarshalAndValidateIdentityDocument(payload, c.awsCaCertPublicKey)
+	c, err := p.getConfig()
```

Comment:
FYI, this check was moved after the Recv() to make the tests more deterministic (failing the RPC before the recv means that sometimes the send will fail on the client side). In practice this shouldn't impact production behavior as the plugins are configured immediately on load before they are used and the config check is wholly defensive with the way the catalog currently works.